apps/prairielearn/elements/pl-file-upload/pl-file-upload.js

```diff
@@ -30,6 +30,8 @@
         });
       }
 
+      this.checkIconColor = options.checkIconColor;
```

Comment:
Copying over my comment from #7536: I view this as a hack and we should get back to standard Bootstrap colors in CSS as soon as we can (aka when we upgrade to Bootstrap v5).